### PR TITLE
fix: CODE spans stuck to words

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/fr/web/javascript/guide/regular_expressions/unicode_property_escapes/index.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Les séquences d'échappement pour les propriétés Unicode permettent de distin
 | `\p{Other_Number} \p{UnicodePropertyName=Other_Number}`               |                                                                                                                  |
 | `\p{Other_Punctuation} \p{UnicodePropertyName=Other_Punctuation}`     |                                                                                                                  |
 | `\p{Paragraph_Separator} \p{UnicodePropertyName=Paragraph_Separator}` |                                                                                                                  |
-| `\p{Private_Use}`Meaning `\p{UnicodePropertyName=Private_Use}`        |                                                                                                                  |
+| `\p{Private_Use}` `\p{UnicodePropertyName=Private_Use}`        |                                                                                                                  |
```
Comment (cw118) [inline files/fr/web/javascript/guide/regular_expressions/unicode_property_escapes/index.md:64]:
```suggestion
| `\p{Private_Use} \p{UnicodePropertyName=Private_Use}`        |                                                                                                                  |
```

File: files/fr/web/svg/element/radialgradient/index.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ html,body,svg { height:100% }
 - [`gradientUnits`](/fr/docs/Web/SVG/Attribute/gradientUnits)
   - : Cet attribut définit le système de coordonnées pour les attributs `cx`, `cy`, `r`, `fx`, `fy`, et `fr`.
     - Type de valeur
-      - : `userSpaceOnUse`|`objectBoundingBox`
+      - : `userSpaceOnUse` | `objectBoundingBox`
```
Comment (cw118) [inline files/fr/web/svg/element/radialgradient/index.md:82]: Will this be fixed on the [English page](https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/radialgradient/index.md?plain=1#L65) as well?

File: files/fr/web/svg/element/radialgradient/index.md [hunk 3]
```diff
@@ -111,7 +111,7 @@ html,body,svg { height:100% }
 - [`srpeadMethod`](/fr/docs/Web/SVG/Attribute/spreadMethod)
   - : Cet attribut indique la façon dont le dégradé se comporte s'il démarre ou finit à l'intérieur des limites de la forme contenant le dégradé.
     - Type de valeur
-      - : `pad`|`reflect`|`repeat`
+      - : `pad` | `reflect` | `repeat`
```
Comment (cw118) [inline files/fr/web/svg/element/radialgradient/index.md:114]: (Same problem on [en-US](https://github.com/mdn/content/blob/main/files/en-us/web/svg/element/radialgradient/index.md?plain=1#L77))

File: files/ru/mdn/tools/kumascript/troubleshooting/index.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ original_slug: MDN/Kuma/Troubleshooting_KumaScript_errors
 - Использование фигурных скобок без намерения вызвать макрос.
   - : Если вам нужно написать \\{ в документе, не вызывая макрос, вы можете экранировать его используя таким образом `\\{`
 - Использование специального символа в параметрах макроса.
-  - : Если вам нужно использовать " или \ внутри параметра макроса, они могут быть экранированы с помощью `\"` и `\\` соответственно 
+  - : Если вам нужно использовать " или \ внутри параметра макроса, они могут быть экранированы с помощью `\"` и `\\` соответственно
```
Comment (cw118) [inline files/ru/mdn/tools/kumascript/troubleshooting/index.md:28]: It seems that this change is unrelated/the wrong locale? I don't really mind keeping it, but the review process might be a little bit slower.

**Review by cw118 — CHANGES_REQUESTED** [review]
Nitpicking a bit and some questions. That being said, thanks again @nschonni !

File: files/fr/web/svg/element/radialgradient/index.md (hunk 2 above)
Comment (nschonni) [reply files/fr/web/svg/element/radialgradient/index.md:82]: Yeah, I'm planning to PR some stuff upstream as well, since I relalized there are probably similar things from the conversion. I'm good reverting this one till later as I didn't change all these occurrences yet

File: files/ru/mdn/tools/kumascript/troubleshooting/index.md (hunk 4 above)
Comment (nschonni) [reply files/ru/mdn/tools/kumascript/troubleshooting/index.md:28]: right, that was unintentional, I'll rebase it out

File: files/fr/web/svg/element/radialgradient/index.md (hunk 2 above)
Comment (nschonni) [reply files/fr/web/svg/element/radialgradient/index.md]: Ended up removing it for now when rebasing out the other file

**Review by cw118 — APPROVED** [review]
This looks good, thank you :)

**Comment by nschonni** [comment]
No problem, and feel free to commit suggestions to my PRs if you want to tweak things to make them easier to land in the future 😄
